src/treq/client.py

```diff
@@ -377,12 +364,10 @@ def _guess_content_type(filename):
 registerAdapter(_from_bytes, bytes, IBodyProducer)
 registerAdapter(_from_file, BytesIO, IBodyProducer)
 
-if not _PY3:
-    from StringIO import StringIO
+if PY2:
     registerAdapter(_from_file, StringIO, IBodyProducer)
```

Comment:
io.StringIO is the wrong StringIO

```suggestion
    registerAdapter(_from_file, six.StringIO, IBodyProducer)
```

Reply:
it's best to
```
import io
import six
```

and then use `io.StringIO` or `six.StringIO` so it's obvious

Reply:
I think it is coming from `six`.  (aren't these `from` imports great?)

Reply:
ah right, that's super confusing

Reply:
@altendky the first hit for `six` uses an idiomatic import: https://github.com/twisted/treq/blob/bea68788c7d2a5d9f8ec15140ec1db20aa26d73a/src/treq/test/local_httpbin/child.py#L13

Reply:
I'd recommend following modernize here and using idiomatic-style imports for both `io` and `six` https://modernize.readthedocs.io/en/latest/fixers.html#unicode_type

Reply:
Over following the existing local practice?  (and that import is off in some other file that i didn't get to touching)